src/data/prayers/resolver.js

```diff
@@ -20,7 +20,7 @@ export default {
   Mutation: {
     addPrayer: (root, args, { dataSources }) => dataSources.Prayer.add(args),
     deletePrayer: (root, { nodeId }, { dataSources }) =>
-      dataSources.Prayer.delete(parseGlobalId(nodeId)),
+      dataSources.Prayer.deletePrayer(parseGlobalId(nodeId).id),
```

Comment:
🔥 Haha. This works for `incrementPrayerCount` because we're pulling `id` as `prayerId` off of the `parseGlobalId(nodeId)`. It just wasn't actually pulling the `id` of the prayer from the parsed object. Nice.